modules/ingress-operator.adoc

```diff
@@ -1,14 +1,33 @@
 // Module included in the following assemblies:
 //
 // * operators/operator-reference.adoc
+// * installing/overview/cluster-capabilities.adoc
+
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
 
 :_mod-docs-content-type: REFERENCE
 [id="ingress-operator_{context}"]
-= Ingress Operator
+ifdef::operator-ref[= Ingress Operator]
+ifdef::cluster-caps[= Ingress Capability]
 
 [discrete]
 == Purpose
 
+ifdef::cluster-caps[]
+The Ingress Operator provides the features for the Ingress capability. Ingress capability is enabled by default.
+
+[IMPORTANT]
+====
+If you set `baselineCapabilitySet` to `None`, you must explicitly enable Ingress capability, because installation of a cluster fails if Ingress capability is disabled.
```

Comment:
```suggestion
If you set the `baselineCapabilitySet` field to `None`, you must explicitly enable the Ingress capability because the installation of a cluster fails if the Ingress capability is disabled.
```

Reply:
:heavy_check_mark: